File: scan-app/app/src/main/java/framework/utils/ImageLoaderUtils.java
package framework.utils;

import android.net.Uri;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.lock.R;

/**
 * by y on 2017/2/17
 */

public class ImageLoaderUtils {

    public static void display(ImageView imageView, Uri url) {
        if (imageView == null) {
            throw new IllegalArgumentException("argument error");
        }
        Glide.with(imageView.getContext()).load(url).error(R.drawable.ic_header).centerCrop().into(imageView);
    }
}

File: sample_lock/src/main/java/framework/utils/ImageLoaderUtils.kt
package framework.utils

import android.net.Uri
import android.widget.ImageView

import com.bumptech.glide.Glide

/**
 * by y on 2017/2/17
 */

object ImageLoaderUtils {

    fun display(imageView: ImageView?, url: Uri) {
        if (imageView == null) {
            throw IllegalArgumentException("argument error")
        }
        Glide.with(imageView.context).load(url).into(imageView)
    }
}
